routers/repo/view.go

```diff
@@ -40,12 +41,33 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 		return
 	}
 
+	query := ctx.Req.URL.Query()
+
+	var start, count int = 0, 100
```

Comment:
I prefer the count could be configured and If we ignored some files we have to give a hint on the UI.